lib/config/index.js

```diff
@@ -42,6 +48,60 @@ const getParser = () => {
                         : assertArrayOfStrings(value, 'commands');
                 }
+            }),
+            nativeElementsSize: section({
+                topToolbarHeight: option({
+                    defaultValue: defaults.nativeElementsSize.topToolbarHeight,
+                    parseEnv: JSON.parse,
+                    parseCli: JSON.parse,
+                    validate: (value, _config, currentNode) => {
+                        if (
+                            (!_.isNull(value) && !_.isNumber(value)) ||
+                            (_.isNull(value) && !isNativeElementsSizeEmpty(currentNode))
+                        ) {
+                            throw new Error(`"topToolbarHeight" must be a number but got ${value === null ? 'null' : typeof value}`);
+                        }
+                    }
+                }),
+                bottomToolbarHeight: option({
+                    defaultValue: defaults.nativeElementsSize.bottomToolbarHeight,
+                    parseEnv: JSON.parse,
+                    parseCli: JSON.parse,
+                    validate: (value, _config, currentNode) => {
+                        if (
+                            (!_.isNull(value) && !_.isNumber(value)) ||
+                            (_.isNull(value) && !isNativeElementsSizeEmpty(currentNode))
+                        ) {
+                            throw new Error(`"bottomToolbarHeight" must be a number but got ${value === null ? 'null' : typeof value}`);
+                        }
+                    }
+                }),
+                webviewHeight: option({
+                    defaultValue: defaults.nativeElementsSize.webviewHeight,
+                    parseEnv: JSON.parse,
+                    parseCli: JSON.parse,
+                    validate: (value, _config, currentNode) => {
+                        if (
+                            (!_.isNull(value) && !_.isNumber(value)) ||
+                            (_.isNull(value) && !isNativeElementsSizeEmpty(currentNode))
+                        ) {
+                            throw new Error(`"webviewHeight" must be a number but got ${value === null ? 'null' : typeof value}`);
+                        }
+                    }
+                }),
+                webviewWidth: option({
+                    defaultValue: defaults.nativeElementsSize.webviewWidth,
+                    parseEnv: JSON.parse,
+                    parseCli: JSON.parse,
+                    validate: (value, _config, currentNode) => {
+                        if (
+                            (!_.isNull(value) && !_.isNumber(value)) ||
+                            (_.isNull(value) && !isNativeElementsSizeEmpty(currentNode))
+                        ) {
+                            throw new Error(`"webviewWidth" must be a number but got ${value === null ? 'null' : typeof value}`);
+                        }
+                    }
+                })
```

Comment:
To my mind these options must looks like:
```
nativeElementsSize: {
  topToolbar: {width, height}, // or 100x200,
  bottomToolbar: {width, height}, // or 100x200,
  webview: {width, height}, // or 100x200,
}
```

This will reduce the number of options and if the width is needed, you will not need to add another option `topToolbarWidth`. What do you think?

Example how we validate `windowSize` in hermione - https://github.com/gemini-testing/hermione/blob/master/src/config/browser-options.js#L258-L274